src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,16 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Drawers {
+  export class ErrorBar extends Drawer {
```

Comment:
`Drawers.LineSegment`? This confusion is part of why I wanted a `Drawers.Path`...